ada_feeding_perception/config/food_on_fork.yaml

```diff
@@ -0,0 +1,19 @@
+# NOTE: You have to change this node name if you change the node name in the launchfile.
+food_on_fork:
+  ros__parameters:
+    # Top-left point's x-val of the bounding box rectangle
+    left_top_corner_x: 297
+    # Top-left point's y-val of the bounding box rectangle
+    left_top_corner_y: 248
+    # Bottom-right point's x-val of the bounding box of the rectangle
+    right_bottom_corner_x: 422
+    # Bottom-right point's y-val of the bounding box of the rectangle
+    right_bottom_corner_y: 332
+    # minimum depth to consider (note that 330 is approx distance to the fork tine)
+    min_dist: 310
+    # maximum depth to consider (note that 330 is approx distance to the fork tine)
+    max_dist: 370
+    # absolute path for the model
+    model_location: "/home/atharva2/atharvak_node_ws/src/ada_feeding/ada_feeding_perception/model/logistic_reg_model.pkl"
```

Comment:
This should not be an absolute path, it should be a relative path, so it works across computers. See how I handle the model path in Food Segmentation. Fundamentally, I pass in an additional param in the launchfile, `model_dir`, that ROS resolves to be the absolute path to the model directory, and then I append that to the relative path passed into the config file. 

(Note that for that to work, you need to rerun `colcon build` any time you change the model, to get it to actually move into the folder where ROS moves all installed files in the workspace.)